.hgignore

```diff
@@ -0,0 +1,30 @@
+^\.servo
+^\.cargo
+^\.servobuild
+^target/
+^ports/android/bin
+^ports/android/libs
+^ports/android/local.properties
+^ports/android/obj
+^python/_virtualenv
+^python/tidy/servo_tidy\.egg-info
+~$
+\.pkl$
+\.pyc$
+\.swp$
+\.swo$
+\.csv$
```

Comment:
Not sure about `git`, but it looks like `hg` can detect those. [Here](https://dxr.mozilla.org/mozilla-central/rev/1d025ac534a6333a8170a59a95a8a3673d4028ee/.gitignore#60) is a similar thing in mozilla-central's `.gitignore` and the [alternative](https://dxr.mozilla.org/mozilla-central/rev/1d025ac534a6333a8170a59a95a8a3673d4028ee/.hgignore#61) in `.hgignore`